include/emp/config/SettingConfig.hpp

```diff
@@ -70,6 +71,18 @@ namespace emp {
                   emp::Ptr<T> _var=nullptr)   ///< Pointer to variable to set (optional)
         : SettingBase(_name, _desc, _flag, _arg), var_ptr(_var) { }
 
+      ~SettingInfo(){if(var_ptr){var_ptr.Delete();}}
+
+      emp::Ptr<SettingBase> Clone() const override {
```

Comment:
Can we just use a std::shared_ptr instead of doing this bookkeeping ourselves? This isn't performance-critical code & would simplify a lot of the work we're doing.

Reply:
Ah, looking more I didn't see how deeply rooted this issue would be in existing code. We'd have to fix a LOT to switch, not worth it.